tools/mcl/mcl.xml

```diff
@@ -1,5 +1,5 @@
 <tool id="mcl_clustering" name="MCL"  version="14.137">
-    <description>A Markov Cluster Algorithm.</description>
+    <description>A Markov Cluster Algorithm</description>
```

Comment:
Yeah, I saw that, and thought the tool depends on it. But I haven't added anything for it to go for mcl.